src/Microsoft.Health.Fhir.TemplateManagement/Constants.cs

```diff
@@ -29,5 +42,7 @@ internal static class Constants
         internal const int TimeOutMilliseconds = 30000;
 
         internal const string OrasFile = "oras";
+
+        internal static IReadOnlyDictionary<DataType, Tuple<string, string>> DefaultTemplateInfo => _defaultTemplateInfo;
```

Comment:
keep constants class for const member only.

Reply:
move default template info from constants.cs to a new model class DefualTemplateInfo